test/applications/k8s/main.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"os"
+	"path"
+	"time"
+
+	nested "github.com/antonfisher/nested-logrus-formatter"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/networkservicemesh/sdk/pkg/tools/debug"
+	"github.com/networkservicemesh/sdk/pkg/tools/grpcutils"
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+	"github.com/networkservicemesh/sdk/pkg/tools/signalctx"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+
+	"github.com/networkservicemesh/cmd-forwarder-sriov/local/sdk-sriov/pkg/tools"
+	"github.com/networkservicemesh/cmd-forwarder-sriov/test/applications/k8s/deviceplugin"
+	"github.com/networkservicemesh/cmd-forwarder-sriov/test/applications/k8s/podresources"
+)
+
+const kubeletSocket = "kubelet.sock"
+
+// Config - configuration for k8s
+type Config struct {
+	DevicePluginPath string `default:"/var/lib/kubelet/device-plugins/" desc:"path to the device plugin directory"`
+	PodResourcesPath string `default:"/var/lib/kubelet/pod-resources/" desc:"path to the pod resources directory"`
+}
+
+func main() {
```

Comment:
Why do we need this application? Can we just use the real application in tests?

Reply:
It is to stub the kubelet API in the docker tests. It is not for testing forwarder -> kubelet interaction but simply for preventing forwarder to fail with _"Oh no, there is no kubelet socket, I can't register, I need to kill myself!"_ in other tests.

Reply:
Can we just start a mock server near the application in the tests? In this case, we'll test real application (main.go) instead of some kind of stub.